tests/integration/test_logcollector/test_log_filter_options/test_ignore_regex.py

```diff
@@ -0,0 +1,357 @@
+t3_configurations = load_configuration_template(t3_configurations_path, t3_configuration_parameters,
+                                                t3_configuration_metadata)
+
+prefix = lc.LOG_COLLECTOR_PREFIX
+local_internal_options = lc.LOGCOLLECTOR_DEFAULT_LOCAL_INTERNAL_OPTIONS
+
+
+# Tests
+@pytest.mark.tier(level=0)
+@pytest.mark.parametrize('new_file_path,', [test_file], ids=[''])
+@pytest.mark.parametrize('configuration, metadata', zip(t1_configurations, t1_configuration_metadata), ids=t1_case_ids)
+def test_ignore_default(configuration, metadata, new_file_path, create_file, truncate_monitored_files,
+                        set_wazuh_configuration, configure_local_internal_options_function, restart_wazuh_function):
+    '''
+    description: Check if logcollector reads or ignores a log according to a regex configured in the ignored tag for a
+                 given log file.
+
+    test_phases:
+        - Set a custom Wazuh configuration.
+        - Restart monitord.
+        - Insert the log message.
+        - Check expected response.
+
+    wazuh_min_version: 4.5.0
+
+    tier: 0
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Wazuh configuration data. Needed for set_wazuh_configuration fixture.
+        - metadata:
+            type: dict
+            brief: Wazuh configuration metadata
+        - new_file_path:
+            type: str
+            brief: path for the log file to be created and deleted after the test.
+        - create_file:
+            type: fixture
+            brief: Create an empty file for logging
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate all the log files and json alerts files before and after the test execution.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set the wazuh configuration according to the configuration data.
+        - configure_local_internal_options:
+            type: fixture
+            brief: Configure the local_internal_options file.
+        - restart_wazuh_function:
+            type: fixture
+            brief: Restart wazuh.
+
+    assertions:
+        - Check that logcollector is analyzing the log file.
+        - Check that logs are ignored when they match with configured regex
+
+    input_description:
+        - The `configuration_ignore_regex_default.yaml` file provides the module configuration for this test.
+        - The `cases_ignore_regex_default` file provides the test cases.
+
+    expected_output:
+        - r".*wazuh-logcollector.*Analizing file: '{file}'.*"
+        - r".*wazuh-logcollector.*DEBUG: Reading syslog '{message}'.*"
+        - r".*wazuh-logcollector.*DEBUG: Ignoring the log line '{message}' due to {tag} config: '{regex}'"
+    '''
+    log = metadata['log_sample']
+    command = f"echo {log}>> {test_file}"
+
+    if sys.platform == 'win32':
+        file = re.escape(test_file)
+    else:
+        file = test_file
```

Comment:
```suggestion
file = re.escape(test_file) if sys.platform == 'win32' else test_file
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3582/commits/b80219b3124d37fc4ca52d3aa63a2c08e03dfe9a